src/workers/autounexile.py

```diff
@@ -0,0 +1,29 @@
+from discord.ext import tasks
+from discord.ext.commands import Bot
+from database import exiles_database
+from services.exile_service import unexile_user
+from settings import get_settings
+from .helper import create_autounexile_embed
+import logging
+
+settings = get_settings()
+logger = logging.getLogger(__name__)
+
+
+@tasks.loop(minutes=1.0)
+async def autounexile_users(self):
+    exiles = exiles_database.get_pending_unexiles()
+
+    for exile in exiles:
+        member = self.get_guild(settings.guild_id).get_member(exile.user_id)
+        if member is not None:
+            async with create_autounexile_embed(
+                self, member, exile.exile_id, exile.end_timestamp
+            ) as autounexile_embed:
+                await unexile_user(autounexile_embed, member)
+        else:
+            logger.info(f"User <@{exile.user_id}> not found in server")
```

Comment:
oops, autopiloted for that one..
for the latter, it's probably better to just use the user id since that's something that can't be changed (plus i don't know if there's necessarily a way to get the username of a person that does not share any servers from just the user id alone)

Reply:
we could do something like user.name but as long as we can map it to the list of users in a server i think it's fine